test/DualTest.jl

```diff
@@ -137,6 +137,14 @@ for N in (0,3), M in (0,4), V in (Int, Float32)
         @test round(FDNUM2) === round(PRIMAL2)
         @test round(NESTED_FDNUM) === round(PRIMAL)
 
+        @test fld(FDNUM, FDNUM2) === fld(PRIMAL, PRIMAL2)
+        @test fld(FDNUM, PRIMAL2) === fld(PRIMAL, PRIMAL2)
+        @test fld(PRIMAL, FDNUM2) === fld(PRIMAL, PRIMAL2)
+
+        @test cld(FDNUM, FDNUM2) === cld(PRIMAL, PRIMAL2)
+        @test cld(FDNUM, PRIMAL2) === cld(PRIMAL, PRIMAL2)
```

Comment:
Did something go weird with the indebting here?

Reply:
I don't really see any weirdness.

Reply:
Yes, @oxinabox, what seems weird?

Reply:
Never mind, probably something off with the rendering on my phone.
It looked like this was at a different indent level to rest of file